wiki/Tournaments/RESC/2025/en.md

```diff
@@ -0,0 +1,162 @@
+---
+tags:
+  - ResCup
+  - Res Cup
+  - Serenity
+  - Regulus
+  - Regulus Revolution
+---
+
+# Resurrection Cup 2025
+
+![Resurrection Cup 2025](img/banner.jpg)
+
+**Resurrection Cup 2025** is a 4v4, double-elimination osu! tournament hosted by ::{ flag=HU }:: [Phreel](https://osu.ppy.sh/users/12840110) and ::{ flag=VN }:: [Hoaq](https://osu.ppy.sh/users/7696512). It is the fourth instalment in the Resurrection Cup series. Resurrection Cup 2025 also introduced a new Resurrection Cup mascot, Regulus.
```

Comment:
where mascot?

Reply:
If you look at the banner, you will see Serenity, the mascot introduced last year, as well as Regulus, the new mascot, on the left and ride sides. While we haven't seen much of her yet, she will make more appearances down the line.